### DISCUSSION TITLE
Cant connect with hotmail.com

### DISCUSSION THREAD

**Discussion by nytram-md** [moralmunky/Home-Assistant-Mail-And-Packages#565] [Q&A] [answered]
Re-installed Mail and Packages and trying to configure

![image](https://user-images.githubusercontent.com/55548608/141813453-b8f073a6-80bd-4533-b347-764dc3f535f1.png)

HA 2021.11.3

Ive tried various hosts for hotmail.com and 993 but to no avail with my email account and password. Email and password works as I double checked and signed out to confirm both.

What have I done wrong ?

Thanks

Martyn

**Comment by firstof9** [comment, marked as answer]
This is [covered in the wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Configuration-and-Email-Settings#hotmailoutlooklive), check the linked help page.

"If you want to add your Outlook.com account to a smart device like a home security camera, you'll need an app password ..."

**Comment by nytram-md** [comment]
Found that you do need an app password.

Thank you

**Comment by jbistis** [comment]
I have the same issue and must not be doing the fix correctly.  I use cloud and see that it is not supported, so I just created an outlook.com user that I will forward to.  I turned on two-step verification and then made a new app password.  I've tried installing the integration Step 1 of 2 with Host outlook.office365.com, Port 993, username is my Outlook email address, and I've tried both passwords, the one I used to create the Outlook account and the app password and neither works.  I get the message "Unable to connect or login to the mail server. Please check the log for details."  The log file has the message "2024-12-03 17:20:41.819 ERROR (MainThread) [custom_components.mail_and_packages.helpers] Error logging into IMAP Server: b'LOGIN failed.'"   What am I doing wrong?

**Reply by firstof9** [reply]
Microsoft's rejecting your login.

**Reply by jbistis** [reply]
Which password do I use? Is it the one I used when I created the Outlook account or the app password I created?  I've tried both, and neither works.  Is there anything else you see that I have wrong?
![outlook-config](https://github.com/user-attachments/assets/de2d92f4-c61f-4f92-a798-4c1ad1484a01)

**Reply by jbistis** [reply]
I'm not sure where the disconnect is, but now, when I try to connect, my Pi crashes. I am an iCloud user, anyhow. Is support for that coming out soon? I saw an issue logged about three weeks ago.

**Reply by firstof9** [reply]
> I am an iCloud user, anyhow. Is support for that coming out soon?

A fix is in the works for iCloud.
